java/client/src/org/openqa/selenium/support/ui/FluentWait.java

```diff
@@ -262,4 +301,33 @@ private Throwable propagateIfNotIgnored(Throwable e) {
   protected RuntimeException timeoutException(String message, Throwable lastException) {
     throw new TimeoutException(message, lastException);
   }
+
+  /**
+   * Converts the {@code TimeUnit} to the equivalent {@code ChronoUnit}.
+   *
+   * This is a backport from Java 9, see https://bugs.openjdk.java.net/browse/JDK-8141452.
+   *
+   * @param timeUnit the TimeUnit to convert
+   * @return the converted equivalent ChronoUnit
+   */
+  private ChronoUnit toChronoUnit(TimeUnit timeUnit) {
+    switch (timeUnit) {
+      case NANOSECONDS:
+        return ChronoUnit.NANOS;
+      case MICROSECONDS:
+        return ChronoUnit.MICROS;
+      case MILLISECONDS:
+        return ChronoUnit.MILLIS;
+      case SECONDS:
+        return ChronoUnit.SECONDS;
+      case MINUTES:
+        return ChronoUnit.MINUTES;
+      case HOURS:
+        return ChronoUnit.HOURS;
+      case DAYS:
+        return ChronoUnit.DAYS;
+      default:
+        throw new AssertionError();
```

Comment:
Typically, this would be an `InvalidArgumentException` instead.

Reply:
this is an exact copy of the method from Java 9, I thought it could simplify future migration to Java 9, but if you wish, I'll change it to `InvalidArgumentException`